test/screenshot/lib/cbt-user-agent.js

```diff
@@ -107,6 +110,20 @@ async function fetchBrowserByApiName(cbtDeviceApiName, cbtBrowserApiName) {
   });
 }
 
+/**
+ * @return {!Array<string>}
+ */
+function getFilteredAliases() {
+  return require('../browser.json').user_agent_aliases.filter((alias) => {
+    const isIncluded =
+      cliArg.includeBrowserPatterns.length === 0 ||
+      cliArg.includeBrowserPatterns.some((pattern) => pattern.test(alias));
+    const isExcluded =
+      cliArg.excludeBrowserPatterns.some((pattern) => pattern.test(alias));
+    return isIncluded && !isExcluded;
```

Comment:
Nice. If I'm understanding this, it allows for complex inclusion/exclusion ("all baseline tests that aren't for button but only on Edge" would be something like `--mdc-include-url=baseline --mdc-exclude-url=button --mdc-include-browser=edge`?)

Reply:
Yes, exactly!

Reply:
😍